hs/app/reach/Main.hs

```diff
@@ -2450,6 +2451,149 @@ whoami = command "whoami" $ info f fullDesc
   where
     f = pure . script $ write [N.text| echo "$whoami'" |]
 
+type DeviceCode = Text
+
+-- https://stackoverflow.com/a/60790430
+newtype SomeValue = SomeValue String
+instance FromJSON SomeValue where
```

Comment:
Bad name

Reply:
I agree. Thanks for reminding me. [55ab671](https://github.com/reach-sh/reach-lang/pull/762/commits/55ab6711358f48de6a1bf8d39f13ba84a7590b28) should address this.